src/snapred/backend/data/Indexer.py

```diff
@@ -129,7 +126,7 @@ def reconcileIndexToFiles(self):
                     f"The following records were expected, but not available on disk: {missingRecords}"
                 )
             else:
-                logger.warn(f"The following records were expected, but not available on disk: {missingRecords}")
+                logger.warning(f"The following records were expected, but not available on disk: {missingRecords}")
```

Comment:
This warning could be improved by asking the user to consult an IS to locate the records or to edit the index